README.md

```diff
@@ -1,2 +1,39 @@
-# scala-base-conf
-Scala Base Conference
+# Scalabase Conf 14 May 2021
```

Comment:
Have we decided this is the way we want to write it, not Scala Base?

Reply:
I have no idea 😂